docs/content/dagster-cloud/developing-testing/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -52,7 +52,7 @@ In this step, you'll add the required GitHub workflow files to your GitHub repos
 
 Copy the following files to your repository:
 
-- [`cloud_workspace.yaml`](https://github.com/dagster-io/dagster-cloud-branch-deployments-quickstart/blob/main/cloud_workspace.yaml)
+- [`cloud_workspace.yaml`](https://github.com/dagster-io/dagster-cloud-branch-deployments-quickstart/blob/main/cloud_workspace.yaml) - **Note**: Save this file as `dagster_cloud.yaml`. This file has been re-named, but not yet reflected in this template repository.
```

Comment:
Note - this repo hasn't been updated with the new name yet. Changing the name in the URL here will throw a 404, so opting to keep it with a clarification.

Reply:
dont worry about it. ben's working on it and we'll do a full pass later